embedded-aerospike-enterprise/src/main/java/com/playtika/testcontainers/aerospike/enterprise/AerospikeEnterpriseConfigurer.java

```diff
@@ -15,26 +14,29 @@ public class AerospikeEnterpriseConfigurer {
     private final AerospikeProperties aerospikeProperties;
     private final AerospikeEnterpriseProperties enterpriseProperties;
 
-    public void configure(GenericContainer<?> aerospikeContainer)  throws IOException, InterruptedException {
+    public void configure(GenericContainer<?> aerospikeContainer) throws IOException, InterruptedException {
         if (aerospikeProperties.getFeatureKey() == null || aerospikeProperties.getFeatureKey().isEmpty()) {
             log.warn("Evaluation feature key file not provided by 'embedded.aerospike.featureKey' property. " +
                     "Pay attention to license details: https://github.com/aerospike/aerospike-server.docker/blob/master/enterprise/ENTERPRISE_LICENSE");
         }
 
-        setupDisallowExpunge(aerospikeContainer);
-    }
-
-    private void setupDisallowExpunge(GenericContainer<?> aerospikeContainer) throws IOException, InterruptedException {
-        if (!enterpriseProperties.isDurableDeletes()) {
-            return;
-        }
-        log.info("Setting up 'disallow-expunge' to true...");
         String namespace = aerospikeProperties.getNamespace();
-        Container.ExecResult result = aerospikeContainer.execInContainer("asadm", "-e",
-                String.format("enable; manage config namespace %s param disallow-expunge to true", namespace));
-        if (result.getStderr().length() > 0) {
-            throw new IllegalStateException("Failed to set up 'disallow-expunge' to true: " + result.getStderr());
+        AsadmCommandExecutor asadmCommandExecutor = new AsadmCommandExecutor(aerospikeContainer);
+
+        /*
+         By default, the value of this metric is 90%, we set it to 100% to prevent stopping writes for the Aerospike
+         Enterprise container during high consumption of system memory. For the Aerospike Community Edition, this metric is not used.
+         Documentation: https://aerospike.com/docs/server/reference/configuration#stop-writes-sys-memory-pct
+        */
+        log.info("Switching off 'stop-writes-sys-memory-pct'... ");
+        asadmCommandExecutor.execute(String.format("manage config namespace %s param stop-writes-sys-memory-pct to 100", namespace));
```

Comment:
Why don't you check execute the result of the command?

Reply:
it is checked in AsadmCommandExecutor